task-list/app/models/task.rb

```diff
@@ -1,2 +1,16 @@
 class Task < ActiveRecord::Base
+  def self.list
+		sorted_tasks = {completed: [], todo: []}
+		Task.all.each do |task|
+			sorted_tasks[:completed] << task unless task.completed_at.nil?
+			sorted_tasks[:todo] << task if task.completed_at.nil?
+		end
+
+    return sorted_tasks
+  end
+
+  def status
+    return "Completed" unless self.completed_at.nil?
```

Comment:
It seems like it would be better to use a "regular" if/else for this logic instead of two separate statements